docs/client/introduction.mdx

```diff
@@ -3,21 +3,60 @@ title: Client SDKs
 slug: /client/introduction
 ---
 
-Statsig Client SDKs evaluate gate, config, experiment checks up front for a particular user, and then serve all checks synchronously from this evaluation.
+Statsig client SDKs enable you to run experiments for user experience improvements in applications running on end-user devices. These applications include web and mobile apps, television and gaming apps, and other consumer electronics apps.  
 
-In general, Client SDKs follow the same pattern:
+As shown in the diagram below, implementing an experiment using a Statsig client SDK consists of three phases:
+1. Initialization
+2. Serving Variants
+3. Recording Exposure Checks & Custom Events
 
-1. Initialize the SDK.  The SDK will reach out to Statsig and evaluate all feature gates, dynamic configs, experiments, autotunes, and layers for the given user
```

Comment:
Its more like:

- initialize
- check an experiment
- log custom events

They dont have to do anything to evaluate variants, and they dont have to record exposure checks manually. So its a lot to introduce at once